src/USER-REAXC/reaxc_types.h

```diff
@@ -39,6 +39,10 @@
 #include <sys/time.h>
 #include "accelerator_kokkos.h"
 
+#include "lammps.h"
+#include "error.h"
+using LAMMPS_NS::Error;
```

Comment:
we particularly do not want `using` anywhere in a header file.

Reply:
I removed my change, but there is another `using` here: https://github.com/lammps/lammps/blob/eedc88eb9b82701740e48903634395e7990b2cf3/src/USER-REAXC/reaxc_types.h#L399

Reply:
need to check about getting rid of that, too. the code in USER-REAXC has been rather "dirty" when it was merged into LAMMPS and caused trouble because of that multiple times. So this is a work-in-progress and every opportunity to clean it up incrementally is welcome. i will sort this one out.

Reply:
the required change is now in PR #1402 as commit 6cb120d691b656e32a5619005041768842683d6b